extensions/azurecore/src/account-provider/providerSettings.ts

```diff
@@ -48,6 +48,11 @@ const publicAzureSettings: ProviderSettings = {
 				endpoint: 'https://vault.azure.net',
 				azureResourceId: AzureResource.AzureKeyVault
 			},
+			azureDevOpsResource: {
```

Comment:
Before this is merged, since we now fully support US Govt, you should fill in the endpoints for AzureDevops on US Govt now